services/monitor/instances/monitor_gcf_env_secrets/instance.yaml

```diff
@@ -0,0 +1,16 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+gcf:
+  triggerTopic: cai-rces-cloudfunctions-CloudFunction
```

Comment:
Would you update solution.yaml to add the related asset type `    - cloudfunctions.googleapis.com/CloudFunction` in BOTH list `resources` and `iampolicies` ?

Reply:
Done